omod/src/main/java/org/openmrs/module/reporting/web/datasets/ManageDatasetsController.java

```diff
@@ -119,4 +147,40 @@ public String saveDataSet(
 
         return "redirect:/module/reporting/definition/manageDefinitions.form?type=org.openmrs.module.reporting.dataset.definition.DataSetDefinition";
     }
+    
```

Comment:
You can eliminate the use of this "AffectedReport" class by utilizing the "DefinitionSummary" class that already exists for this purpose.  It allows representation of a lighter-weight, non-hibernate-managed "Definition", and includes "uuid", "name", "description", and "type" properties that are simple strings.